src/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 	<title>Travel with Your Friends</title>
 	<link href="/Users/yang/final-project-boston-bucket-list/src/css/_layout.scss" rel="stylesheet" type="text/css">
+	<script src='/Users/yang/final-project-boston-bucket-list/src/js/scripts.js'></script>
```

Comment:
You don't have to link these - they'll be linked automatically for you when the page builds! Go ahead and remove them. (Also, you're the only one that can see `/Users/yang/` anyway - that's a folder on your computer, so I won't be able to see it when I go to the webpage.)